mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping

```diff
@@ -0,0 +1,55 @@
+CLASS net/minecraft/unmapped/C_vykgtyww net/minecraft/client/util/telemetry/TelemetryProperty
+	FIELD f_apuzxcvp exportKey Ljava/lang/String;
+	FIELD f_avrsqphd exporter Lnet/minecraft/unmapped/C_vykgtyww$C_bqxexqea;
+	FIELD f_ccowaobp WORLD_LOAD_TIME Lnet/minecraft/unmapped/C_vykgtyww;
+	FIELD f_dhokegej id Ljava/lang/String;
+	FIELD f_iridotjb codec Lcom/mojang/serialization/Codec;
+	FIELD f_lvqaugyd DATE_TIME_FORMATTER Ljava/time/format/DateTimeFormatter;
+	METHOD <init> (Ljava/lang/String;Ljava/lang/String;Lcom/mojang/serialization/Codec;Lnet/minecraft/unmapped/C_vykgtyww$C_bqxexqea;)V
+		ARG 1 id
+		ARG 2 exportKey
+		ARG 3 codec
+		ARG 4 exporter
+	METHOD m_agomzdpf createLongList (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_gbrbzzts createInteger (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_hpowlaaz export (Lnet/minecraft/unmapped/C_tupaamrk;Lcom/mojang/authlib/minecraft/TelemetryPropertyContainer;)V
+		ARG 1 propertyMap
+		ARG 2 container
+	METHOD m_jadmvviu createUuid (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_jvqgfioi createString (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_maatndmi createBoolean (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_pmddrquw codec ()Lcom/mojang/serialization/Codec;
+	METHOD m_tkjnpann exportKey ()Ljava/lang/String;
+	METHOD m_ulasainp create (Ljava/lang/String;Ljava/lang/String;Lcom/mojang/serialization/Codec;Lnet/minecraft/unmapped/C_vykgtyww$C_bqxexqea;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+		ARG 2 codec
+		ARG 3 exporter
+	METHOD m_usjtckev text ()Lnet/minecraft/unmapped/C_npqneive;
+	METHOD m_ypyjfwfa id ()Ljava/lang/String;
+	CLASS C_afeghidc ServerType
+		FIELD f_hfcrjewu key Ljava/lang/String;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
+			ARG 3 key
+	CLASS C_bqxexqea Exporter
+		METHOD apply export (Lcom/mojang/authlib/minecraft/TelemetryPropertyContainer;Ljava/lang/String;Ljava/lang/Object;)V
```

Comment:
```suggestion
		METHOD apply (Lcom/mojang/authlib/minecraft/TelemetryPropertyContainer;Ljava/lang/String;Ljava/lang/Object;)V
```

Reply:
oh no, the only deobfuscated rename I could remember was `log()`